lib/rex/post/meterpreter/client_core.rb

```diff
@@ -9,7 +9,6 @@
 # Used to generate a reflective DLL when migrating. This is yet another
 # argument for moving the meterpreter client into the Msf namespace.
 # URI uuid and checksum stuff
```

Comment:
Just confirming; Do these comments need moved / removed / updated?

Reply:
I think we should go one step further and go along with what the comment said, move the meterpreter stuff into `Msf` it does not belong in rex :)

Reply:
wait, these are two different comments
the second one can go because the require is disappearing

The first one can stay but I still think it's a good idea